client/src/contexts/user/UserContextProvider.js

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import { SIGN_UP, LOGIN, LOGOUT } from "../types";
+
+const initialState = {
+    isAuthenticated: false,
+    profile: {
+        _id: "",
+        email: "",
+        favoriteCuisine: [],
+        isChef: false,
+        primaryAddress: {},
+        profileImage: "",
+    },
+    token: null,
```

Comment:
We don't want to store the token in our app itself, we'll just let the browser handle it